.cargo/mutants.toml

```diff
@@ -11,7 +11,9 @@ exclude_re = [
     "deserialize", # Skip serde mutation tests
     "serde_details::<impl de::Visitor<'_>", # Skip serde mutation tests
     "Iterator", # Mutating operations in an iterator can result in an infinite loop
-    "impl encoding::Decodable", # Mutant replacing Default::default() is equivalent to returning new()
+    "<impl encoding::Decodable for .*>::decoder", # Mutant replacing Default::default() is equivalent to returning new()
+    "<impl encoding::Decoder for .*>::min_bytes_needed", # Function is just a cast for optimization.
```

Comment:
What do you mean by this comment 'is just a cast'? Also, give how unsure I was when writing impls for this method it makes me nervous to exclude it.

Reply:
FTR I just read https://github.com/rust-bitcoin/rust-bitcoin/issues/5088#issuecomment-3369057117 after reviewing so I see where you got the optimisation bit.

Reply:
Since this is approved I'll leave the comment as is. But if you have a better version I'll add it next time. Probably next monday, going by the last few weeks..

Reply:
Heh, oops, I read this as "just a case of optimization".

But we can change the comment in a followup.